### PR TITLE
Jaholm/cop 32 carrier class

### PR DESCRIPTION
**Pull request by Holmness**
Not completly done, but functional

Needs method for generating Carriers dynamicly

### CODE REVIEW

File: train_simulation/Railway.py [hunk 1]
```diff
@@ -87,10 +88,10 @@ def __str__(self):
     def from_json(self, json_dct):
         Connection(json_dct['station A'],
                    json_dct['station B'],
-                   json_dct['distance']),
-        Connection(json_dct['station B'],
-                   json_dct['station A'],
                    json_dct['distance'])
+        # Connection(json_dct['station B'],
```
Comment (KuroHaka) [inline train_simulation/Railway.py]: Don't comment this, it won't work

File: train_simulation/Simulation.py [hunk 2]
```diff
@@ -43,9 +43,10 @@
 sys.path.append(PROJECT_ROOT)
 dirname = os.path.dirname(__file__)
 
-from train_simulation.Moving import Train
+from train_simulation.Rail_Transport import Train, Carrier
```
Comment (KuroHaka) [inline train_simulation/Simulation.py:46]: I like this name 🤣

**Review by KuroHaka — CHANGES_REQUESTED** [review]

File: train_simulation/Railway.py (hunk 1 above)
Comment (Holmness) [reply train_simulation/Railway.py]: Yes it does

File: train_simulation/Rail_Transport.py [hunk 3]
```diff
@@ -316,15 +313,265 @@ def getUID(self):
...
-# train[0].boardPassengers(350)
-# train[0].disembarkPassengers(50)
-# train[0].moveTo('Nørreport')
-# train[0].printInformation()
-# train[0].decelerate()
-# train[0].arriveAt('Nørreport')
-# print()
-# train[0].printInformation()
\ No newline at end of file
+class Carrier(Moving):
```
Comment (KuroHaka) [inline train_simulation/Rail_Transport.py]: You can remove the Moving abstract class. We are not using pygame anymore

**Review by KuroHaka — CHANGES_REQUESTED** [review]

File: train_simulation/Simulation.py (hunk 2 above)
Comment (Holmness) [reply train_simulation/Simulation.py:46]: Kek

**Review by KuroHaka — APPROVED** [review]

**Pull request merged**